yo @tsconfig/node-ts, your version is all fucked up and shit

When I run `npm install` in the repo root, I get this error output:

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @tsconfig/node-ts@^1.0.4.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

And sure enough, this version makes no gosh darn sense. The latest stable version is `23.6.1`.

--- a/package.json
+++ b/package.json
@@ -11,6 +11,6 @@
     "lint": "npm run lint --workspaces"
   },
   "devDependencies": {
-    "typescript": "^5.0.0"
+    "typescript": "^5.7.2"
   }
 }

--- a/packages/bullshit-detector/package.json
+++ b/packages/bullshit-detector/package.json
@@ -25,16 +25,16 @@
   "devDependencies": {
     "@jest/globals": "^29.0.0",
     "@tsconfig/node22": "^22.0.0",
-    "@tsconfig/node-ts": "^1.0.4",
+    "@tsconfig/node-ts": "^23.6.1",
     "@types/jest": "^29.0.0",
-    "@types/node": "^20.0.0",
-    "@typescript-eslint/eslint-plugin": "^6.0.0",
-    "@typescript-eslint/parser": "^6.0.0",
-    "eslint": "^8.0.0",
+    "@types/node": "^22.0.0",
+    "@typescript-eslint/eslint-plugin": "^8.0.0",
+    "@typescript-eslint/parser": "^8.0.0",
+    "eslint": "^9.0.0",
     "jest": "^29.0.0",
     "jest-environment-node": "^29.0.0",
     "ts-jest": "^29.0.0",
-    "typescript": "5.9.2"
+    "typescript": "^5.7.2"
   },
   "dependencies": {
     "openai": "^4.0.0",

